resources/startPage.html.template

```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src {{ cspSource }} file:; script-src 'nonce-{{ nonce }}'; font-src {{ codiconsFontUri }}; style-src {{ cspSource }} {{ codiconsStyleUri }} 'nonce-{{ nonce }}';" />
+    <link href="{{ codiconsStyleUri }}" rel="stylesheet" />
+    <title>Docker - Get Started</title>
+
+    <style nonce="{{ nonce }}">
+        #titleBar {
+            align-items: center;
+        }
+
+        #titleBar h1 {
+            margin-block-start: .2em;
+            margin-block-end: .2em;
+        }
+
+        .icon .codicon {
+            font-size: 70px;
+            padding: 5px;
+        }
+
+        .icon:hover {
+            color: var(--vscode-editor-foreground);
+        }
+
+        #dockerIcon {
+            width: 70px;
+            padding: 5px;
+        }
+
+        #dockerIcon path {
+            fill: var(--vscode-editor-foreground);
+        }
+
+        .row {
+            display: flex;
+            flex-direction: row;
+            justify-content: flex-start;
+            align-items: flex-start;
+        }
+
+        .row>* {

```

Comment:
`.row > *` for extra readability

Reply:
I tried but VSCode immediately reformatted it back 😭 

I'm sure there's a setting but for this I don't know it's worth disabling, other CSS formatting features are good.

Reply:
Of course 🙄